tasks/pipeline-dbconf.yml

```diff
@@ -36,6 +36,7 @@
     encoding: "utf8"
     collation: "utf8_unicode_ci"
   register: "createdb_result"
+  when: 'archivematica_src_am_database_host == "localhost"'
```

Comment:
I think it would be good to add a note in the README saying that mysql manual database/user creation is required when the host is not localhost